Cargo.toml

```diff
@@ -69,28 +71,33 @@ rand_core = "0.5.0"
 ring = "0.16.20"
 jsonwebtoken = "8.2.0"
 data-encoding = "2.3.3"
-chrono = "0.4.23"
+chrono = { version = "0.4.23", features = ["serde"] }
 derive_more = "0.99.17"
 futures = "0.3.26"
 async-tungstenite = {version = "0.20.0", features = ["tokio", "tokio-native-tls"]}
 dashmap = "5.4.0"
 
-relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", tag = "v0.17.0", features = ["cacao"]}
-relay_client = { git = "https://github.com/WalletConnect/WalletConnectRust.git", tag = "v0.17.0" }
+relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", rev = "5f4dd3cbf4a67e40c47503706f8e0ae8d8bdd435", features = ["cacao"]}
+relay_client = { git = "https://github.com/WalletConnect/WalletConnectRust.git", rev = "5f4dd3cbf4a67e40c47503706f8e0ae8d8bdd435" }
 x25519-dalek = { version = "2.0.0-rc.2" , features = ["static_secrets"]}
 hkdf = "0.12.3"
 sha2 = "0.10.6"
 uuid = { version= "1.3.1", features = ["serde", "v4"] }
 
 reqwest = { version = "0.11", features = ["json"] }
 anyhow = "1.0.71"
+pnet_datalink = "0.33.0"
+ipnet = "2.8.0"
+once_cell = "1.18.0"
 
 [dev-dependencies]
 test-context = "0.1"
 
 [build-dependencies]
 build-info-build = "0.0"
 
+[features]
+analytics = []
```

Comment:
Feature is nice, is there any case where:
1. Someone runs this themself?
2. Analytics won't be enabled?

If not might be simpler to not include the flag?

Reply:
True, removed it as Cast will only be hosted with them anyways